core/include/debug.h

```diff
@@ -14,7 +14,7 @@
  * @brief       Debug-header
  *
  * @details     If ENABLE_DEBUG is set, before this header is included,
- *              ::DEBUG will print out to stdout, otherwise do nothing
+ *              ::DEBUG_FUNC will print out to stdout, otherwise do nothing
```

Comment:
Yes, you're right. This change is utter nonsense. The correct fix would be to add documentation for `DEBUG`.